cockroachcloud/authorization.md

```diff
@@ -1,44 +1,158 @@
+
+We discourage further use of the Legacy Org Developer role as that may have a broader set of entitlements than what you may want to assign to non-admins. Same would apply to Legacy Org Admin once the new Org Admin (to be added in a future release) role is available. The legacy roles will be deprecated after the updated CockroachDB Cloud authorization model is generally available to all customers, and will be completely removed six months after that.
+
+
+### Does the CockroachDB Cloud authorization model apply similarly to both service accounts and human users in a CockroachDB Cloud organization?
+
+Yes, for service accounts created after release v23.1. Older, legacy service accounts use an older authorization model. See [Service Accounts](#service-accounts).
+
+### Could I assign a cluster level role to a few users such that they have the relevant entitlements on all clusters in the CockroachDB Cloud organization?
+
+Yes, an admin could assign a cluster level role like Cluster Admin, Cluster Operator Reader and Writer (to be added in a future release) or Cluster Developer on the entire CockroachDB DB Cloud organization or on one or more specific clusters. There are two scopes in the authorization model - organization and clusters, with organization being the parent, and clusters being the children in the hierarchy. So if an admin assigns cluster level roles at the organization scope, they are automatically applicable on all clusters in the CockroachDB DB Cloud organization. Such access should be granted only to users who need to work with all clusters.
+
+### What happens if an admin removes all role assignments for a particular user? Is that user removed from the CockroachDB Cloud organization?
+
+When all role assignments have been removed for a user, they still retain the Org Member role which is added at the time when a user is added to the CockroachDB Cloud organization. The user is not automatically removed at that point. To remove a user from the CockroachDB Cloud organization, the admin should take the specific remove action.
+
+### Which roles are allowed to add and remove users, and manage their role assignments in a CockroachDB Cloud organization?
+
+Only users with the Org Admin role are allowed to add and remove users at the organization level. Users with the Cluster Admin role are allowed to manage role assignments at the cluster level. If a user has been assigned both Org Admin and Cluster Admin roles at the organization scope (which is true for the initial user by default), they can manage role assignments at both organization and cluster scopes.
```

Comment:
```suggestion
Users with the Org Admin (legacy) role are allowed to manage users & roles at both the organization and the cluster levels. Users with the Cluster Admin role are only allowed to manage role assignments at the cluster level.
```

Reply:
@laurenbarker @gary-crl @pawalt - Can you plz validate this edit suggestion? Thanks.

Reply:
The edit suggestion is accurate 👍